dev-docs/source/howto/install.rst

```diff
@@ -103,8 +103,16 @@ To address the error we need to mark that dataset as a proxy to another URL::
 Run the server
 --------------
 
+The server needs a JSON Web Key Set (JWKS) to do authorization.
```

Comment:
Does the test token have an expiry? If so, maybe add a warning about that.

Reply:
Done.